cabot/cabotapp/tasks.py

```diff
@@ -91,17 +96,27 @@ def update_all_services():
 
 
 @task(ignore_result=True)
-def update_shifts():
+def update_shifts_and_problems():
```

Comment:
When we have multiple Cabot workers running, we need to be careful about backward incompatibility for celery. If we have 2 Cabot machines A and B and we deploy A first, wait a minute, then deploy B with this code, we could run into a situation where 
- machine A is deployed
- machine A schedules the task update_shifts_and_problems
- it's routed to machine B, which hasn't been deployed to yet, so doesn't have the code for update_shifts_and_problems, so the task fails

There's a similar possibility with any renamed task + the new send_schedule_problems_email task.

A way to get around this problem is by doing a multipart deploy:
 - deploy code that adds the new task but don't delete the old one
 - deploy code that starts sending off the new task
 - deploy code that deletes the old task

May not be super necessary for this task specifically since it runs so infrequently (and isn't critical), but I think we should start being more careful about this going forward (for reasons we can discuss offline since the repo is open source--have some stories about celery backwards incompatibility)